pallene/to_ir.lua

```diff
@@ -83,6 +83,7 @@ function ToIR:init()
     self.func_stack     = {} -- list of function to_ir.FuncInfo
     self.call_exps      = {} -- { ast.Exp.CallFunc }
     self.dsts_of_call   = {} -- { ast.Exp => { var_id } }
+    self.toplevel_funcs = {} -- { ast.FuncStat.FuncStat }
```

Comment:
This is a small hack for now. Since we will be getting rid of `ir.Value.Function` down the line anyway.
This table separates toplevel functions from non-toplevel functions.